crates/primitives-traits/src/block/body.rs

```diff
@@ -29,8 +45,17 @@ pub trait BlockBody:
     + MaybeArbitrary
 {
     /// Ordered list of signed transactions as committed in block.
-    type Transaction: Transaction;
+    type Transaction: SignedTransaction;
```

Comment:
can we do this separately?

Reply:
https://github.com/paradigmxyz/reth/pull/12903